flowmachine/tests/test_random.py

```
@@ -233,9 +234,11 @@ def test_system_rows_fail_with_inheritance():
     """
     Test whether the system row method fails if the subscriber queries for random rows on a parent table.
     """
-    with pytest.raises(ValueError):
-        df = Table(name="events.calls").random_sample(
-            size=8, sampling_method="system_rows"
+    with pytest.raises(PreFlightFailedException):
+        df = (
```

Ruff

Local variable `df` is assigned to but never used

Remove assignment to unused variable `df`

(F841)